nexus/src/authz/roles.rs

```diff
@@ -132,19 +134,20 @@ pub async fn load_roles_for_resource(
     roleset: &mut RoleSet,
 ) -> Result<(), Error> {
     // If the user is authenticated ...
-    if let Some(actor_id) = authn.actor() {
+    if let Some(actor) = authn.actor() {
         // ... then fetch all the roles for this user that are associated with
         // this resource.
         trace!(opctx.log, "loading roles";
-            "actor_id" => actor_id.id.to_string(),
+            "actor" => ?actor,
```

Comment:
Good question.  The reason I changed this is to capture other information about the actor (like the silo user id).  This might differ for different variants of `Actor`.  How about if I `impl Debug for Actor` explicitly with a comment about not including anything sensitive?  That way we'll hopefully catch ourselves from leaking something private, but people can still freely use `Debug` output for objects like this?

Reply:
Done in 8c625082a705523338886c1d08eed7f63d4fcbcb.